Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -0,0 +1,457 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think the method DataGridCellAutomationPeer.GetAutomationControlTypeCore() should be updated to account for this new type of column.

Reply:
I did look at that method but I wasn't sure if it should be the AutomationControlType for the view or edit mode that is returned. I therefore didn't change anything.

I have now put it in as AutomationControlType.ComboBox. Please advise if it is incorrect.